src/components/DendogramVis.js

```diff
@@ -0,0 +1,138 @@
+import React, { Component } from 'react';
+import * as d3 from 'd3';
+import { useD3 } from '../hooks/useD3';
+import '../App.css';
+import { keys } from '@material-ui/core/styles/createBreakpoints';
+
+// var data = {"children":
+// [{"name":"boss1","children":[{"name":"mister_a","colname":"level3", "color": "red"},{"name":"mister_b","colname":"level3"},{"name":"mister_c","colname":"level3"},{"name":"mister_d","colname":"level3"}],"colname":"level2"},{"name":"boss2","children":[{"name":"mister_e","colname":"level3"},{"name":"mister_f","colname":"level3"},{"name":"mister_g","colname":"level3"},{"name":"mister_h","colname":"level3"}],"colname":"level2"}],"name":"CEO"};
+  
+
+// Purpose: ANnotate with some meta-data useful for visualization
+function enrichDendogram(node, color) {
+
+    node['children'] = [];
+    if ("left" in node) {
+        let transformLeftNode = enrichDendogram(node['left'], node['color'] && node['color'] != 'black' ? node['color'] : color);
+        node['children'].push(transformLeftNode);
+    }
+    if ('right' in node) {
+        let transformRightNode = enrichDendogram(node['right'], node['color'] && node['color'] != 'black' ? node['color'] : color);
+        node['children'].push(transformRightNode);
+    }
+    node['color'] = node['color'] && node['color'] != 'black' ? node['color'] : color;
+    return node;
+}
+
+function DendogramVis({dendogram}) {
+
+    d3.select("#dendogram").html("");
+    const ref = useD3(
+        (svg) => {
+            var root = {"color": "white", 'children': []}
+            for (var key in dendogram) {
+                let subtreeRoot = dendogram[key];
+                var subtree = enrichDendogram(subtreeRoot, "black");
+                root['children'].push(subtree);
+            }
+            var data = root;
+            // set the dimensions and margins of the graph
+            var width = 460
+            var height = 660
```

Comment:
In the future, need to fine-tune this width and height of dendogram, should be adaptive and dynamic to number of nodes. C